code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -40,6 +40,49 @@
 	max_heat_protection_temperature = GLOVES_MAX_HEAT_PROTECTION_TEMPERATURE
 	species_fit = list(VOX_SHAPED)
 
+/obj/item/clothing/gloves/swat/operator
+	name = "operator gloves"
+	desc = "Once you touch down in the LZ at the FOB, locate the IEDs and don't put up with any FNGs."
+
+/obj/item/clothing/gloves/swat/operator/examine(mob/user)
+	..()
+	if(locate(/obj/item/weapon/implant/loyalty) in user)
+		to_chat(user,"<span class='info'>These gloves can be used to convey messages to other loyalty implanted crew. Use an open hand on yourself while wearing them.</span>")
+
+/obj/item/clothing/gloves/swat/operator/Touch(var/atom/A, mob/user, proximity)
+	if(A == user && !user.incapacitated())
+		if(locate(/obj/item/weapon/implant/loyalty) in user)
+			var/list/choices = list(
+				list("Stick together!", "radial_group"),
+				list("Split up!", "radial_split"),
+				list("Wait here!", "radial_wait"),
+				list("Busy, cover!", "radial_busy")
```

Comment:
I appear to have accidentally doubled up on an icon name here, so this isn't using the busy hand symbol for the radial, needs unique icon name